examples/README.md

```diff
@@ -18,5 +18,51 @@ Start with the examples under the basic folder in this order:
 4. http-random-user
 5. bmi-naive
 
-This will get you introduced to one main concept with each example. Make sure
-to reach for each example's README file for more context.
+There're corresponding examples between difficulty levels such as bmi-naive with bmi-typescript or animation with animated-letters.
+
+
+## Methods and libraries used by examples
+
+#### Basic
+
+**bmi-naive** Buggy input field as result of reuse as component that will sync with sister instances for lack of isolation, written in rxjs (legacy) as apposed to xstream.
+
+**checkbox** What would be considered a controlled input in react. Method of input handling that permits change from both the user and program.
+
+**counter** Counts up or down on respective button press, state/model accumulation.
+
+**hellow-world** Uses input provided by user to change view ie "Hello, textYouProvided".
+
+**http-random-user** Example that uses typescript to make http requests. Passes events generated from one driver to another after transformation. Typescript provides hints in supporting IDEs and prevents somer errors that xstream doesn't handle well.
+
+**jsx-seconds-elapsed** Timer rendered from jsx file that uses html tags.
+
+#### Intermediate
+
+**animation** Moves a square between 3 points in archs.
+
+**bmi-typescript** Correct version of bmi in xstream with typescript (reusable components through isolation).
+
+**hello-lastname** Constructs and validates full name from two separate inputs in typescript.
+
+**http-search-github** Searches for github repositories filtered by text from input. Debounces http calls, and selects the correct call result out of noisy driver.
+
+**tsx-seconds-elapsed** Typescript and jsx usage demonstration by way of timer.
+
+#### Advanced
+
+**animated-letters** Will toggle showing letters on keypress, with font size change. State handling without @cycle/state.
+
+**autocomplete-search** 200 lines of code implementing autocomplete field for wikipedia. 
```

Comment:
wikipedia => Wikipedia

Mentioning the lines of code is unnecessary in my opinion